src/icons/device/NanoXBanner.js

```diff
@@ -6,7 +6,7 @@ export default ({ size = 30, ...p }: { size: number }) => (
   <svg viewBox="0 0 6 16" height={size} width={size} {...p}>
     <defs>
       <path
-        id="a"
+        id="NanoXBanner-a"
```

Comment:
to me, for all ids you have added we need to actually also template it with a random id (per component mount).

one trick would be to use a ref.

```
const id = useRef(_globalId++)

id={`${id}_NanoXBanner-a`}
```

Reply:
feel free to improve upon this! :D